apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -100,6 +101,14 @@ function DisputableActionStatus({ vote }) {
             trail.
           </Info>
         </Item>
+        <Item>
```

Comment:
This `Item` is adding an additional margin on this last element, making the space feel a little larger than it should.

![image](https://user-images.githubusercontent.com/11708259/86477502-76635780-bd40-11ea-9ce7-377cfec4c1df.png)

We can omit the margin for the last item, either with `:not(:last-child)` or a `Item + Item` selection in the styling (I think the second option is preferred)